flocker/node/agents/blockdevice.py

```diff
@@ -869,12 +878,12 @@ def run(self, deployer):
         )
         device = api.get_device_path(volume.blockdevice_id)
 
-        # This duplicates CreateFilesystem now.
-        check_output(["mkfs", "-t", "ext4", device.path])
+        create = CreateFilesystem(volume=volume, filesystem=u"ext4")
```

Comment:
I don't think the change is valid.  The action discovery code is simply buggy.  It was passing by accident before because there weren't other actions being logged to confuse it.  Now there are other actions to confuse it.  It's failing because it finds an unrelated mount operation in the log.  If the resize code somehow stopped logging its mount operation, the new version of the assertion would still pass and that would be incorrect.

So I think it would be better to revert the assertion change until we want to spend time correctly fixing that helper.